lib/Steps/OpenSentry.ts

```diff
@@ -48,7 +48,7 @@ export class OpenSentry extends BaseStep {
 
       const urlToOpen = urlObj.toString();
 
-      opn(urlToOpen).catch(() => {
+      opn(urlToOpen, { wait: false }).catch(() => {
```

Comment:
good catch! I never would have thought that this is `true` by default (looks like they flipped [from `true`](https://www.npmjs.com/package/opn/v/5.4.0#wait) [to `false`](https://www.npmjs.com/package/opn#wait) in opn 6.0.0 but we still use 5.4.0)